api/pr/controllers/pr.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  async create(ctx) {
+    const {
+      state: { user },
+      request: { body },
+    } = ctx;
+
+    const pr = { ...body, user: user.id };
+
+    const entity = await strapi.services.pr.create(pr);
+    console.log(entity, 'entity');
```

Comment:
please remove this in the next PR .. if you want to log, then use `strapi.log.debug(....)` or `.error` etc.. if it was just for testing then remove it